docker-compose.yml

```
@@ -26,11 +26,17 @@ services:
     environment:
       - DATABASE_URL=postgres://${POSTGRESQL_USER:-zipline}:${POSTGRESQL_PASSWORD}@postgresql:5432/${POSTGRESQL_DB:-zipline}
     depends_on:
-      - postgresql
+      postgresql:
+        condition: service_healthy
     volumes:
       - './uploads:/zipline/uploads'
       - './public:/zipline/public'
       - './themes:/zipline/themes'
+    healthcheck:
+      test: ['CMD', 'wget', '-q', '--spider', 'http://localhost:3000/api/healthcheck']
```

using `localhost` here won't resolve since most of the time Zipline binds to `0.0.0.0` in docker, so it would have to be `zipline` or `0.0.0.0` for the host.

```suggestion
      test: ['CMD', 'wget', '-q', '--spider', 'http://0.0.0.0:3000/api/healthcheck']
```